src/XRay/XRayJSON.ts

```diff
@@ -149,21 +166,39 @@ namespace XRayJSON {
         );
     }
 
+    function checkRegionalExposure (
+        regionalExposureArray?: unknown
+    ): regionalExposureArray is Array<RegionalExposure> {
+        return (
+            !!regionalExposureArray &&
+            typeof regionalExposureArray === 'object' &&
+            regionalExposureArray instanceof Array &&
+            (
+                regionalExposureArray &&
+                regionalExposureArray.length === 0 ||
+                isRegionalExposure(regionalExposureArray[0])
+            )
+        );
+    }
+
```

Comment:
I do not think that this function is necessary, just do the `globalStockSector` part in `isBreakdowns`.